backend/src/main/java/com/exadel/carinsurance/service/implementation/AuthService.java

```diff
@@ -88,6 +105,13 @@ public ResponseEntity login( @RequestBody AuthRequestEntity request ) {
     UserEntity user = ( UserEntity ) authentication.getPrincipal();
     String jwtToken = jwtService.generateToken( userEmail );
 
+    Cookie cookie = new Cookie( "token", jwtToken );
+    cookie.setMaxAge( 86400 );
+    cookie.setPath( "/" );
+
+    httpServletResponse.addCookie( cookie );
+    httpServletResponse.setHeader( HttpHeaders.ACCESS_CONTROL_EXPOSE_HEADERS, "Set-Cookie" );

```

Comment:
You don't need to have `httpServletResponse` to set Cookie. Cookie is one of the headers.
Please refer to `org.springframework.http.ResponseCookie`, `org.springframework.http.HttpHeaders`, `org.springframework.http.ResponseEntity.headers()`

Reply:
Done